app/models/concerns/paper_trailable.rb

```diff
@@ -8,15 +8,15 @@ module PaperTrailable
   end
 
   def created_by
-    versions.first&.user
+    versions.last&.user
```

Comment:
I cannot tell the difference between asc and desc, which is embarrassing. The scope orders these by id desc, so the most recent id is the most recent change; created by would thereby be the first.